packages/yoastseo/OVERVIEW.md

```diff
@@ -0,0 +1,141 @@
+# YoastSEO.js Overview
+
+YoastSEO.js is a text analysis and SEO assessment library that helps improve content readability and search engine optimization. This document provides an overview of the main concepts and components.
+
+## Domain Glossary
+
+### Core Concepts
+
+- **[Paper](./GLOSSARY.md#paper)**: A value object representing the text content to be analyzed, including metadata like title, meta description, keyword, etc.
+- **[Assessment](./GLOSSARY.md#assessment)**: An individual analysis that evaluates a specific aspect of content (e.g., keyword density, sentence length)
+- **[Assessor](./GLOSSARY.md#assessor)**: A collection of assessments that work together to analyze content from a specific angle (SEO, readability, etc.)
+- **[Researcher](./GLOSSARY.md#researcher)**: Performs linguistic research on text content (e.g., sentence detection, word counting)
+- **[Score](./GLOSSARY.md#score)**: A numeric value (0-100) indicating how well content performs for a specific assessment
```

Comment:
```suggestion
- **[Score](./GLOSSARY.md#score)**: A numeric value (0-90) indicating how well content performs for a specific assessment
```

Reply:
Again, see example above of one of the spec tests that seems to contradict this change.

Reply:
Yep, my bad!